### PR TITLE
[Mac] Use stub mdtool to run tests.

### PR DESCRIPTION
**Pull request by Therzok**
This fixes locating native subversion libs provided by apple in 64bit.

### CODE REVIEW

**Comment by Therzok** [comment]
cc @alanmcgovern

File: version-checks [hunk 1]
```diff
@@ -17,8 +17,8 @@ DEP[0]=md-addins
 DEP_NAME[0]=MDADDINS
 DEP_PATH[0]=${top_srcdir}/../md-addins
 DEP_MODULE[0]=git@github.com:xamarin/md-addins.git
-DEP_NEEDED_VERSION[0]=dba061cf4f3a16f5384faabcf42d082c31503975
-DEP_BRANCH_AND_REMOTE[0]="master origin/master"
+DEP_NEEDED_VERSION[0]=0254d8cd02bc23e8124777fc6a31d3b4e6cd807d
+DEP_BRANCH_AND_REMOTE[0]="stub-mac-tests origin/stub-mac-tests"
```
Comment (DavidKarlas) [inline version-checks]: Change this back to master before merging PR I guess...

**Comment by Therzok** [comment]
Oh, this PR is irrelevant. I just wanted to do fewer clicks to setup a build lane. I manually cherry-picked the commits to master.

**Comment by alanmcgovern** [comment]
:+1:

**Pull request closed without merging**